test/geo/t_io.clj

```diff
@@ -1,9 +1,14 @@
+(def wkb-hex "00000000030000000100000005C0518027525460AA403E007C84B5DCC6C0518027525460AA403E0068DB8BAC71C051801BDA5119CE403E0068DB8BAC71C051801BDA5119CE403E007C84B5DCC6C0518027525460AA403E007C84B5DCC6")
+(def ewkb-hex-wgs84 "0020000003000010E60000000100000005C0518027525460AA403E007C84B5DCC6C0518027525460AA403E0068DB8BAC71C051801BDA5119CE403E0068DB8BAC71C051801BDA5119CE403E007C84B5DCC6C0518027525460AA403E007C84B5DCC6")
+(def wkb-2-hex "00000000030000000100000019C05D9AA4FEC7483740410748DCF001DDC05D9AA54A2FCA834041074874966BEDC05D9AA7CF462C3340410744F7DB6700C05D9AA86ECD473E404107441B4FD3A0C05D9AC2D7FC35CB4041071F8E14EC8DC05D9AC2F86FEAC44041071FD93BC96BC05D9ACBCFE5D7EB404107345BC3A32BC05D9AC753BE8EA24041073A871AC7EEC05D9AC75D3578344041073A9A66DD10C05D9AC6B6EB509F4041073B7F1113F6C05D9AC6AD5119E44041073B6C003FB1C05D9AC4578063DF4041073EA23845A9C05D9AC464DC96AB4041073EBCDDC95CC05D9AC3BA0479D74041073FA7C93A5BC05D9AC3ACCBBA234041073F8D237737C05D9AC1B67B5B724041074242C2FED7C05D9ABF5E512ACB404107457BFD8B9DC05D9ABCCE62E7BF4041074052A3D6D4C05D9ABB7F0B51AE404107421FAC6307C05D9ABA28D088AE4041073F6E483838C05D9AAD827B96DA40410750D649A664C05D9AAD228B96F8404107515A48598FC05D9AACA1ED5ACC404107520B4B3DB8C05D9AAACC52644E4041075491743BC3C05D9AA4FEC7483740410748DCF001DD")
+(def ewkb-2-hex-wgs84 "0020000003000010E60000000100000019C05D9AA4FEC7483740410748DCF001DDC05D9AA54A2FCA834041074874966BEDC05D9AA7CF462C3340410744F7DB6700C05D9AA86ECD473E404107441B4FD3A0C05D9AC2D7FC35CB4041071F8E14EC8DC05D9AC2F86FEAC44041071FD93BC96BC05D9ACBCFE5D7EB404107345BC3A32BC05D9AC753BE8EA24041073A871AC7EEC05D9AC75D3578344041073A9A66DD10C05D9AC6B6EB509F4041073B7F1113F6C05D9AC6AD5119E44041073B6C003FB1C05D9AC4578063DF4041073EA23845A9C05D9AC464DC96AB4041073EBCDDC95CC05D9AC3BA0479D74041073FA7C93A5BC05D9AC3ACCBBA234041073F8D237737C05D9AC1B67B5B724041074242C2FED7C05D9ABF5E512ACB404107457BFD8B9DC05D9ABCCE62E7BF4041074052A3D6D4C05D9ABB7F0B51AE404107421FAC6307C05D9ABA28D088AE4041073F6E483838C05D9AAD827B96DA40410750D649A664C05D9AAD228B96F8404107515A48598FC05D9AACA1ED5ACC404107520B4B3DB8C05D9AAACC52644E4041075491743BC3C05D9AA4FEC7483740410748DCF001DD")
```

Comment:
@willcohen I realize you didn't start the trend of putting these examples inline but now that we're adding so many more of them I'd like to move them to separate files. Ideally we can put each one in its own file under `test/resources` and then just slurp those files into these definitions.

Reply:
Will do!